.gitignore

```diff
@@ -90,6 +90,7 @@ out
 # Nuxt.js build / generate output
 .nuxt
 dist
+main.js
```

Comment:
why do you need  this?
it should be ignored inside "dist" or public.
wherever you build to

Reply:
Im building to public because Im serving the public folder from the server